app/models/user.rb

```diff
@@ -5,4 +5,12 @@ class User < ApplicationRecord
          :recoverable, :rememberable, :validatable
   has_many :recipes
   has_many :foods
+
+  validates :name, presence: true, length: { minimum: 3 }
+
```

Comment:
Good job adding the validation for the name. However, the validation is causing some issues when I try to register with a new user. In the sign-up form, you have just the email and password files, and since the user name should be present as you specify in the `User` model the ` ActiveRecord`  is not creating a new user. I kindly suggest adding a field for the user name in the sign-up form